modules/client-configuration/pages/registration-bootstrap.adoc

```diff
@@ -43,42 +43,34 @@ Attempting to register openSUSE Leap 15 or SLE 15 systems using Python 2 boot
 
 == Create a Bootstrap Script
 
-This procedure describes how to generate a bootstrap script.
+You can use the {productname} {webui} to create an editable bootstrap script.
 
 .Procedure: Creating a Bootstrap Script
 . In the {productname} {webui}, navigate to menu:Admin[Manager Configuration > Bootstrap Script].
 . In the [guimenu]``SUSE Manager Configuration - Bootstrap`` dialog, uncheck the [guimenu]``Bootstrap using Salt`` checkbox if you are installing a traditional client.
 For Salt clients, leave it checked.
-Use default settings and click the btn:[Update] button.
-+
+. The required fields are pre-populated with values derived from previous installation steps.
+For details on each setting, see xref:reference:admin:bootstrap-script.adoc[].
+. Click btn:[Update] create the script.
+. The bootstrap script is generated and stored on the server in the [path]``/srv/www/htdocs/pub/bootstrap`` directory.
+Alternatively, you can access the bootstrap script over HTTPS.
+Replace ``example.com`` with the host name of your {productname} Server:
```

Comment:
I see you downcased this.  Nevertheless, this
```suggestion
Replace ``EXAMPLE.COM`` with the host name of your {productname} Server:
```
Even if you don't accept it we should talk about it.  Just to avoid changing those things back and forth ;)

Reply:
Ah yeah, I was uncertain what we'd agreed. I'll take it to email.

Reply:
So far, the overwhelming consensus is to use angle brackets. I'll update accordingly.